htlcswitch/link_isolated_test.go

```diff
@@ -0,0 +1,268 @@
+package htlcswitch
+
+import (
+	"crypto/sha256"
+	"testing"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lntypes"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+type linkTestContext struct {
```

Comment:
😍

Reply:
Yes, every pr should have a small clean up component...